components/script/dom/window.rs

```diff
@@ -788,13 +788,11 @@ impl WindowMethods for Window {
                     let window = this.root();
                     let document = window.Document();
                     // https://html.spec.whatwg.org/multipage/#closing-browsing-contexts
-                    // Step 1, prompt to unload.
+                    // Step 3, prompt to unload.
```

Comment:
It's probably worth comment on what happened to step 1 and 2 here. :)

Reply:
done in f3a32f480a8afeaa1731712df8ee879a9822d04f